libkbfs/ops.go

```diff
@@ -993,59 +993,62 @@ func (ro *rekeyOp) String() string {
 	return "rekey"
 }
 
-func (ro *rekeyOp) CheckConflict(renamer ConflictRenamer, mergedOp op,
+func (ro *rekeyOp) checkConflict(renamer ConflictRenamer, mergedOp op,
 	isFile bool) (crAction, error) {
 	return nil, nil
 }
 
-func (ro *rekeyOp) GetDefaultAction(mergedPath path) crAction {
+func (ro *rekeyOp) getDefaultAction(mergedPath path) crAction {
 	return nil
 }
 
-// gcOp is an op that represents garbage-collecting the history of a
+// GCOp is an op that represents garbage-collecting the history of a
 // folder (which may involve unreferencing blocks that previously held
 // operation lists.  It may contain unref blocks before it is added to
 // the metadata ops list.
-type gcOp struct {
+type GCOp struct {
```

Comment:
You could add an `op.IsGC()` interface method if you'd rather not export the type. It's a little strange to export this one and not the others, but I don't feel too strongly about it.  Fine with it if you prefer this way.